test/test_api/test_api.py

```diff
@@ -177,6 +179,13 @@ def test_tabular_classification(openml_id, resampling_strategy, backend):
     score = estimator.score(y_pred, y_test)
     assert 'accuracy' in score
 
+    # check incumbent config and results
+    incumbent_config, incumbent_results = estimator.get_incumbent_results()
+    assert isinstance(incumbent_config, Configuration)
```

Comment:
Do you think you can create a unit test, like a separate one to test the `include_traditional`?

What we can do is mock.patch the evaluator.run() method to always return a desired value and quickly compare that everything is working as expected?

For example, the tae evaluator run could return:

- time.time() as cost, so we would expect that the best configuration is for sure a lgbm which is the first thing that is ran
- Then without the include, it should be a pytorch mode

What do you think?

Reply:
Can you tell me how to just set cost to time.time(). As the TAE.run returns status, cost, runtime, additional_run_info, how can I set cost as time and maintain the other info?

Reply:
Todo a patch please follow [this](https://docs.python.org/3/library/unittest.mock.html)

So what we do is that we set as return value a tuple like:
```
eval_patch.return_value = (StatusType.SUCCESS, time.time(), {})
```

We can also provide a tae function [here](https://github.com/automl/Auto-PyTorch/blob/a4e08e297dc298b53d5962f0f4b4e55b5e9ded8c/autoPyTorch/optimizer/smbo.py#L254)
 I think... maybe it is easier?